src/app/components/LeftBanner.module.css

```diff
@@ -0,0 +1,83 @@
+.bannerWrap {
+    background-color: #20365F;
```

Comment:
To Consider - this color is used several times in the project, I would define it as a variable in the globals.css file